sky/clouds/cloud.py

```diff
@@ -498,14 +501,24 @@ def check_cluster_name_is_valid(cls, cluster_name: str) -> None:
 
     @classmethod
     def check_disk_tier_enabled(cls, instance_type: str,
-                                disk_tier: str) -> None:
+                                disk_tier: resources_utils.DiskTier) -> None:
         """Errors out if the disk tier is not supported by the cloud provider.
 
         Raises:
             exceptions.NotSupportedError: If the disk tier is not supported.
         """
         raise NotImplementedError
 
+    @classmethod
+    def translate_disk_tier(
+        cls, disk_tier: Optional[resources_utils.DiskTier]
+    ) -> resources_utils.DiskTier:
+        if disk_tier is None:
+            return cls._DEFAULT_DISK_TIER
+        if disk_tier == resources_utils.DiskTier.BEST:
+            return cls._BEST_DISK_TIER
+        return disk_tier
```

Comment:
What does this mean when a cloud does not support `CUSTOM_DISK_TIER`?

Reply:
It just use default disk tier since it is 'the best among all choices' (you only have one choice)